src/main/java/edu/rpi/legup/history/AutoCaseRuleCommand.java

```diff
@@ -24,6 +24,8 @@ public class AutoCaseRuleCommand extends PuzzleCommand {
 
     private List<TreeTransition> caseTrans;
 
+    private static final int MAX_CASES = 5;
```

Comment:
I'm not sure if this is something we want. Would this break any puzzle that we implement in the future that have the possibility to generate more than 5 cases?

Reply:
It was requested by Prof Bram so that we would not be generating many of cases that cannot all be seen in the 'tree view'. It can be changed to another number if desired.

Reply:
Ok, if Bram said that then we can leave that as is.